### PR TITLE
CSS Transitions Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
animation21

### CODE REVIEW

File: css-transitions/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  cursor: pointer;
+  transform: scale(1);
+  transition: 0.5s ease;
+}
+
+.card > img {
+  width: 100%;
+}
+
+.custom-button:hover {
```
Comment (robgmerrill) [inline css-transitions/styles.css:48]: For code reading clarity I would often have the pseudoclass listed in order immediately after the element it is being used on. For example: 

```
.custom-button {
}

.custom-button:hover {
}
```

Same goes for `.card` and `.card:hover`.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Left some feedback on when pseudoclasses should appear in the sorce order of your CSS.

**Comment by Nathan-Chon** [comment]
animation21
Resubmitted changes to code!

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**